src/main/java/cl/transbank/pos/utils/TbkReturn.java

```diff
@@ -8,47 +8,45 @@
 
 package cl.transbank.pos.utils;
 
-public final class TbkReturn {
-  public final static TbkReturn TBK_OK = new TbkReturn("TBK_OK", TransbankWrapJNI.TBK_OK_get());
-  public final static TbkReturn TBK_NOK = new TbkReturn("TBK_NOK", TransbankWrapJNI.TBK_NOK_get());
+public enum TbkReturn {
```

Comment:
Same as above, since this file is created by Swig changes will be lost upon Wrapper update.